client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -326,16 +332,19 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
      * @param edit
      */
     public async setEditMode(edit: boolean): Promise<void> {
-        if (!this.newUser && edit) {
-            this._isUserInScope = await this.userService.hasScopeManagePerms(this._userId!);
-        }
-
         this.isEditingSubject.next(edit);
 
         // case: abort creation of a new user
         if (this.newUser && !edit) {
             this.goToAllUsers();
         }
+
+        if (edit) {
+            await this.modelRequestService.waitSubscriptionReady(PARTICIPANT_DETAIL_SUBSCRIPTION);
+            setTimeout(() => (this._userFormLoaded = true), 1000);
```

Comment:
I think you could solve the flicker by waiting for `updateEditable` and `waitSubscriptionReady` to be resolved instead of using the timeout.

Reply:
So basically you add to the `loadUserById`:
```
this.modelRequestService.waitSubscriptionReady(...).then(async () => {
    await this.updateEditable();
    this._userFormLoaded = true;
})
```

Reply:
This doesn't solve the flickering.